src/plugins/world_stats/WorldStats.cc

```diff
@@ -57,6 +57,9 @@ namespace plugins
 
     /// \brief Holds iterations
     public: QString iterations;
+
+    public: std::optional<double> simTimeDelayed;
+    public: std::optional<double> realTimeDelayed;
```

Comment:
add doxy comments?

Reply:
Done in https://github.com/gazebosim/gz-gui/pull/529/commits/09fd320ae80869c9cbb2948fd1f4382b5229cbb6